gapis/service/path/path.go

```diff
@@ -630,6 +636,10 @@ func (n *Command) ResourcesAfter(ids []*ID) *MultiResourceData {
 	}
 }
 
+func (n *Command) FramebufferAttachmentsAfter(ids []*ID) *FramebufferAttachments {
```

Comment:
what are the ids for?

Reply:
Holdover from something I was trying a while back. To be honest, this function isn't necessary, and I am thinking of deleting it.